src/claudeExplainPatch.js

```diff
@@ -0,0 +1,64 @@
+import Anthropic from '@anthropic-ai/sdk';
+
+export default async function explainPatch({patchBody, prnum, owner, repo,
+  githubToken,
+  model = 'claude-3-opus-20240229',
+  system = `
+You are an expert software engineer reviewing a pull request on Github. Lines that start with "+" have been added, lines that start with "-" have been deleted. Use markdown for formatting your review.
+
+Desired format:
+### Description
+<description_of_PR> // How does this PR change the codebase? What is the motivation for this change?
+
+### Changes
+<list_of_changes> // Describe the main changes in the PR, organizing them by filename
+
+### Security Hotspots
+<list_of_security_hotspots> // Describe locations for possible vulnerabilities in the change, order by risk
+\n`,
+  max_tokens = 2048,
+  temperature = 1,
+  top_p = 1,
+  debug = false}) {
+  if (!patchBody) {
```

Comment:
this was necessary for me to get the local test to work as described in the readme (following the openai example), but i'm guessing that is just because the readme instructions are missing something.